.github/workflows/pull.yml

```diff
@@ -33,24 +34,27 @@ jobs:
         with:
           node-version: 20
 
+      - name: Install
+        run: npm ci
+
+      - name: Verify Quality (type checking, linting, format checking, tests)
+        run: |
+          npm run verify
+  
       - name: "Authenticate to Google Cloud"
+        if: ${{ github.event.pull_request.draft == false }}
```

Comment:
you should be able to put it above, with the other `if` on L26 .. which, per [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif), should be like
```
if: ${{ ! github.event.pull_request.head.repo.fork }}
```

Reply:
e.g. (not sure if this will work)
```
if: ${{ !github.event.pull_request.head.repo.fork && !github.event.pull_request.draft }}
```

Reply:
can you change the ordering back too .. so that the setup (GCloud stuff is out of the way before we start doing npm tasks ... while you are here, we don't need to split lines for single commands (old stuff) e.g.
```
run: npm run verify
```

Reply:
@algreasley sure.  It make sense to set the condition at the jobs level if we want to skip all the jobs. my initial assumption was that we wanted to still perform npm install and npm verify but these checks are also done on branch level so we are covered ✅